process-create-data-repo-request

```
@@ -0,0 +1,253 @@
+# Functions
+# --------------------------------------------------------------------------------
+
+
+#
+# Searches in the comments if there is a comment made from  the given users  that
+# matches the given pattern. It returns a list with the matched comments.
+#
+def search_in_comments(comments, user_logins, pattern, first_line):
+    found_comments = []
+    requested_comment_bodies = [c.body for c in comments if c.user.login in user_logins]
+    for body in requested_comment_bodies:
+        examined_str = body
+        if first_line:
+            examined_str = str(
+                body.encode("ascii", "ignore").decode().split("\n")[0].strip("\n\t\r ")
+            )
+
+        if examined_str == pattern:
+            found_comments.append(body)
+            continue
+
+        if re.match(pattern, examined_str):
+            found_comments.append(body)
+
+    return found_comments
+
+
+#
+# posts a message to the issue in github
+# if dry-run is selected it doesn't post the message and just prints it
+# if you set checkIfRepeated to False, if will not check if the message has already been written.
+#
+def post_message(issue, msg, comments, checkIfRepeated=True):
+    if checkIfRepeated and search_in_comments(comments, ["cmsbuild"], msg, False):
+        print("Message already in the thread: \n %s" % msg)
+        return
+
+    if opts.dryRun:
+        print("Not posting message (dry-run):\n %s" % msg)
+        return
+
+    print("Posting message:\n %s" % msg)
+    issue.create_comment(msg)
+
+
+#
+# reads the comments and gets returns the status of the issue
+#
+def get_issue_status(issue):
+    global labels
+    labels = [label.name for label in issue.get_labels()]
+    print("Issue Label: ", labels)
+
+    if NEW_ISSUE in labels:
+        return NEW_ISSUE
+    if any(label.endswith("-pending") for label in labels):
+        return PENDING_APPROVAL
+    if FULLY_SIGNED in labels:
+        return FULLY_SIGNED
```

@iarspider , what happens is non of these condition match? I would suggest to remove the first condition and just return `NEW_ISSUE` at the end of this function if no pending or fully singed labels are found

---

Before the script will silently ignore the issue. Now it will still ignore the issue, but will also print a message.